### ISSUE TITLE
Support HTML 5 input types in text box.

### ISSUE THREAD

**Issue by nehashri** [getgauge/taiko#608]
All HTML 5 input type is not supported well. For example, input type of `search` is not supported.

**Steps to reproduce:**
* Execute the below script
```
const { openBrowser, goto, toRightOf, textBox, into, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser();
        await goto("thetrainline.com");
        await write("London",into(textBox(toRightOf("From"))));
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
* The script errors out with `Error: Element focused is not writable`.

**Comment by nehashri** [comment]
Some of the html 5 input types that need to be supported by text box include `text`, `number`, `email`, `password`, `search`, `tel`, `url`.
Currently, `text` and `password` are supported.

**Comment by nehashri** [comment]
Fix available with 2a3f39e971d0a455720d2e0462975736a99e4b33

**Comment by Debashis9012** [comment]
I have verified the above issue for html5. The below html5 types can be handled by ```textBox``` api:
```text, number, email, password, search, tel, url ```
Based on the observations this issue has been found fixed. 
Commit version: ```2a3f39e971d0a455720d2e0462975736a99e4b33```
